pkg/signup/service/signup_service_test.go

```diff
@@ -247,8 +247,10 @@ func (s *TestSignupServiceSuite) TestGetSignupFailsWithNotFoundThenOtherError()
 		return nil, errors2.NewInternalError(errors.New("something quite unfortunate happened"), "something bad")
 	}
 
+	c, _ := gin.CreateTestContext(httptest.NewRecorder())
```

Comment:
Let's use `nil` instead of the test context at least in some tests to make sure it doesn't fail if the context is nil. Or rather use it only in the tests which test the cases when the context is expected.

Reply:
I've added a specific test just for this case.